docs/proposals/algorithms/lifelong-learning/Unknown_Task_Recognition_Algorithm_Reproduction_based_on_Lifelong_Learning_of_Ianvs.md

```diff
@@ -0,0 +1,208 @@
+
+Lifelong learning has five key characteristics.
+
+1. a process of continuous learning.
+2. the accumulation and retention of knowledge in the knowledge base.
+3. the ability to use accumulated learned knowledge to aid future learning
+4. the ability to discover new tasks.
+5. the ability to learn while working.
+
+Relying on the lifelong learning system built by KubeEdge+Sedna+Ianvs distributed collaborative AI joint inference framework, the core task of this project is to complete the unknown task identification algorithm module and embed it in the framework, with the aim of equipping the system with the ability to discover new tasks.
+
+Traditional machine learning performs test set inference by training known samples, whose knowledge is limited, and the resulting models cannot effectively identify unknown samples in new classes, which will be treated as known samples. In a real production environment, it is difficult to guarantee that the training set contains samples from all classes. If the unknown class samples cannot be identified, the accuracy and confidence of the model will be greatly affected, and the cost consumed for model improvement is incalculable. This project aims to reproduce the algorithm of the CVPR2021 paper "Learning placeholders for open-set recognition". The paper proposes placeholders that mimic the emergence of new classes, thus helping to transform closed training into open training to accomplish recognition of unknown classes of data.
+
+In this project, the algorithm is packaged as a python callable module and embedded in the lib library of Ianvs' lifelong learning testing system. The algorithm developer does not need to develop additional algorithms for unknown task recognition and can directly test the performance of the currently developed algorithms in combination with the dataset and testing environment provided by Ianvs. At the same time, Ianvs provides local and cloud-based algorithm performance rankings for developers to facilitate the exchange of lifelong machine learning researchers and thus promote the development of the lifelong learning research field.
+
+
+
+## 2 Goals
+
+1. Reproducing the algorithm from the CVPR2021 paper "Learning placeholders for open-set recognition".
+2. Achieving an accuracy of 0.9 for unknown task recognition.
+3. Making unknown category recognition modules according to Ianvs architecture and contextual relationship to store in the lib library, so that the algorithm can be used on demand.
+
```

Comment:
Add test report in goals https://github.com/kubeedge/ianvs/pull/26#discussion_r947521553